tests/end-to-end/backup-restore-test/deploy/chart/backup-test/values.yaml

```diff
@@ -6,4 +6,4 @@ nameOverride: backup-test
 image:
   name: backup-test
   dir: pr/tests
-  tag: PR-4893
```

Comment:
Please rebase the PR #5044 from master. I've also merged some changes to the backup tests and right now `PR-5044` image does not include them.

Reply:
@suleymanakbas91  #5044 rebased, running tests now